### PR TITLE
Update definition of issuance date

### PR DESCRIPTION
**Pull request by David-Chadwick**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/620.html" title="Last updated on May 13, 2019, 6:16 PM UTC (0ae3195)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/620/5577f77...0ae3195.html" title="Last updated on May 13, 2019, 6:16 PM UTC (0ae3195)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1476,9 +1476,9 @@ <h3>Issuance Date</h3>
 A <a>credential</a> MUST have an <code>issuanceDate</code> <a>property</a>. The
 value of the <code>issuanceDate</code> <a>property</a> MUST be a string value of
 an [[!RFC3339]] combined date and time string representing the date and time
-the <a>credential</a> was issued. Note that this date represents the earliest
+the <a>credential</a> becomes valid. Note that this date represents the earliest
 date when the information associated with the <code>credentialSubject</code>
```
Comment (brentzundel) [inline index.html]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -1476,9 +1476,9 @@ <h3>Issuance Date</h3>
 A <a>credential</a> MUST have an <code>issuanceDate</code> <a>property</a>. The
 value of the <code>issuanceDate</code> <a>property</a> MUST be a string value of
 an [[!RFC3339]] combined date and time string representing the date and time
-the <a>credential</a> was issued. Note that this date represents the earliest
+the <a>credential</a> becomes valid. Note that this date represents the earliest
```
Comment (brentzundel) [inline index.html]: Perhaps:
> the <a>credential</a> becomes valid, which could be a date and time in the future.

Then we don't need the redundant:
> Note that this represents . . .

Comment (David-Chadwick) [reply index.html]: @brentzundel. The original text talked about both the VC and the credential subject. So I have kept this for now, unless people think it is unnecessary.

Comment (David-Chadwick) [reply index.html]: @brentzundel . Take a look at the new Diff and see if you like it now

Comment (brentzundel) [reply index.html]: I like it better. The different sentence constructions help to better clarify that they are talking about two different things. 
I wonder if the sentence starting with "Note" should actually be a note, set apart as such in the text.

**Pull request merged**